### ISSUE TITLE
Non-text elements should have alt text

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1061]
This includes any non-decorative non-image (i.e. img tag) elements.  Examples include, but are not limited to, interactive elements devoid of text, important placeholder svgs, or other icon driven components.
